### PR TITLE
559 live streaming standard is missing

### LINKED ISSUE #559: Live streaming standard is missing
How to use live streaming standard is missing from the docs. It gives some info in the overview but thats it.
It is also missing from the pricing page (it was on vercel 12 days ago)

### PR DESCRIPTION
**Pull request by saudsami**
closes #559

### CODE REVIEW

File: shared/video-sdk/reference/_pricing.mdx [hunk 1]
```diff
@@ -91,7 +93,11 @@ In a live-streaming channel, Agora charges for audio and video service usage bas
 
 - **Audience**:
 
-  - **Low latency audience member**: A user with the audience user role and the low latency user level is an audience member. The price for audio and video minutes are:
+    When you join a channel as audience, the prices for audio and video minutes depend on the selected latency level.
+
+  - **Low latency audience member in <Vg k="ILSS"/>**
+  
+  The prices of audio and video minutes for a user with the audience user role and low latency level are:
```
Comment (billy-the-fish) [inline shared/video-sdk/reference/_pricing.mdx:100]: Can you sort out the indenting please.

File: shared/video-sdk/reference/_pricing.mdx [hunk 2]
```diff
@@ -91,7 +93,11 @@ In a live-streaming channel, Agora charges for audio and video service usage bas
 
 - **Audience**:
 
-  - **Low latency audience member**: A user with the audience user role and the low latency user level is an audience member. The price for audio and video minutes are:
+    When you join a channel as audience, the prices for audio and video minutes depend on the selected latency level.
+
+  - **Low latency audience member in <Vg k="ILSS"/>**
```
Comment (billy-the-fish) [inline shared/video-sdk/reference/_pricing.mdx:98]: ber -  \<Vg k="ILSS"/>

File: shared/video-sdk/reference/_pricing.mdx [hunk 3]
```diff
@@ -101,7 +107,9 @@ In a live-streaming channel, Agora charges for audio and video service usage bas
     | Video 2K      | 7.99                       |
     | Video 2K+     | 17.99                       |
 
-  - **Ultra low latency audience**: A user with the audience user role and `audienceLatencyLevel` set to  `AUDIENCE_LATENCY_LEVEL_ULTRA_LOW_LATENCY` is charged at the same rate as the host.
+  - **Ultra low latency audience member with <Vg k="ILSP"/>**
```
Comment (billy-the-fish) [inline shared/video-sdk/reference/_pricing.mdx:110]: member - \<Vg k="ILSP"/>

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Nearly there.

**Comment by billy-the-fish** [comment]
A bit rude to ask, but while you are at it, I round a random GRE on the following page: https://docs.agora.io/en/flexible-classroom/get-started. Can you remove it please.

File: shared/video-sdk/reference/_pricing.mdx (hunk 1 above)
Comment (saudsami) [reply shared/video-sdk/reference/_pricing.mdx:100]: fixed

**Comment by saudsami** [comment]
> A bit rude to ask, but while you are at it, I round a random GRE on the following page: https://docs.agora.io/en/flexible-classroom/get-started. Can you remove it please.

done

**Pull request merged**